### ISSUE TITLE
Update to bitflags 0.5.

### ISSUE THREAD

**Issue by Ms2ger** [servo/servo#10077]
And remove it from the list of exceptions in `python/tidy.py`, to which I added it in #10076.

**Comment by saurvs** [comment]
I'd like to do this.

**Comment by Ms2ger** [comment]
Go for it. I'd suggest you comment here with to the PRs you create on other repositories.

**Comment by saurvs** [comment]
Just to clarify, every Rust repo part of the Servo org needs to use the new version of bitflags?

**Comment by Ms2ger** [comment]
Each crate that we depend on, here's a list:
```
./ports/geckolib/Cargo.lock:1: 
Version 0.3.3:
		selectors
		style
		util

./ports/geckolib/Cargo.lock:1: 
Version 0.4.0:
		geckoservo

./ports/gonk/Cargo.lock:1: 
Version 0.3.3:
		cocoa
		devtools_traits
		gfx
		layout
		msg
		png
		script
		selectors
		servo
		style
		util
		wayland-client
		wayland-kbd
		websocket
		xml-rs

./ports/gonk/Cargo.lock:1: 
Version 0.4.0:
		openssl

./components/servo/Cargo.lock:1: 
Version 0.3.3:
		servo
		cocoa
		devtools_traits
		gfx
		layout
		msg
		png
		script
		selectors
		style
		util
		wayland-client
		wayland-kbd
		websocket
		xml-rs

./components/servo/Cargo.lock:1: 
Version 0.4.0:
		glutin_app
		openssl
```

**Comment by saurvs** [comment]
Alright.

**Comment by mbrubeck** [comment]
Fixed by #11191.
